### PR TITLE
Share filter and copy URL buttons

### LINKED ISSUE #36: Add sharable filters functionality
What about a button appears at the bottom of filters when filters are applied? A button that says:
"Copy URL for this filter"

Clicking that could copy a URL like the following:
https://turkiyereliefdao.com?cryptocurrencies=Polkadot

When such a URL is loaded, the query parameters must be deleted and the state better is persisted (#35) on page refreshes.

Another button next to it could say:
"Reset filter"

Advantage: 
 * Relevant subsets of options could be shared with networks, resulting in better targeting and conversion.
 
Disadvantages:
 *Harder to implement than most of the other issues i.e. opportunity cost.
 
 [1] Need to create persistent state management.  Even more work. Could be delegated to another issue. The disadvantage of this

### PR DESCRIPTION
**Pull request by nhestrompia**
Created the buttons according to issue #36

### CODE REVIEW

**Comment by hantuzun** [comment]
Is it possible to get these buttons to fit into the same line? I suggest using smaller buttons: https://tailwindui.com/components/application-ui/elements/buttons#component-8a4b7248253ad4c9ee892c655d7ff5ec

"Reset Filter" and "Copy URL for this filter" are inconsistent.

What about: 

"Share This Filter" and "Reset Filter" in this order?

The popup text could be: "Copied the URL for the current filter"

I also suggest using the following [heroicon](https://heroicons.com/) for the reset button:
Screen Shot 2023-02-13 at 06 27 31

**Comment by nhestrompia** [comment]
I tried both on the same row at the beginning but since suggested title was long for URL button they seemed odd so i moved one of them to another row. Will update the code

**Comment by hantuzun** [comment]
Thanks @nhestrompia.

I considered other icon options and I like this one the most now:

Screen Shot 2023-02-13 at 06 38 34

**Comment by hantuzun** [comment]
I'll be reviewing this later today, need to get some sleep...

**Comment by hantuzun** [comment]
You could also push the change I commented on at https://github.com/neu-fi/turkiye-relief-dao/pull/69 to https://github.com/neu-fi/turkiye-relief-dao/tree/51-break-org-card-into-functions if you'd like to make our job easier by merging the two commits.

**Comment by nhestrompia** [comment]
Okay i will use this icon for the reset button. Will check that issue before i push these changes, if i can will do that

**Comment by hantuzun** [comment]
Thanks, @nhestrompia.

**Comment by nhestrompia** [comment]
Sorry for late update, just had a chance. Layout changes to column at some breakpoint because the buttons didn't fit as they were.
Let me know if any more updates needed on this

File: app/Donations.tsx [hunk 1]
```diff
@@ -409,9 +483,65 @@ export default function Organizations() {
                 <h3 className="sr-only">Categories</h3>
 
                 {filters.map((section) => createFilterElement(section))}
-                {sortedOrganizations.length !== organizations.length && (
-                  <p className="text-gray-400 text-center mt-4">
-                    Displaying {sortedOrganizations.length} of{" "}
+                {filters !== initialFilters ? (
```
Comment (oytuncoban) [inline app/Donations.tsx]: Everything is flawless, thank you, @nhestrompia!

Only thing is that, the intention here is to hide reset and copy buttons when there is no filters are set. However, although the user resets the filters, this check returns true and still displays the buttons on the page.

If this behavior is fixed, we are good to go.

**Review by oytuncoban — CHANGES_REQUESTED** [review]

File: app/Donations.tsx (hunk 1 above)
Comment (nhestrompia) [reply app/Donations.tsx]: Fixed this problem. Let me know if more adjustments needed

Comment (oytuncoban) [reply app/Donations.tsx]: Thank you @nhestrompia!

LGTM, merging now.

**Review by oytuncoban — APPROVED** [review]
LGTM!

**Pull request merged**